src/read.jl

```diff
@@ -18,8 +18,8 @@ end
 function Base.tryparse(::Type{RecordingID}, raw::AbstractString)
     s = split(raw, ' ', keepempty=false)
     length(s) == 5 || return
-    first(s) == "Startdate" || return
-    _, start_raw, admin_raw, tech_raw, equip_raw = s
+    popfirst!(s) == "Startdate" || return
+    start_raw, admin_raw, tech_raw, equip_raw = s
```

Comment:
Unless there's something I'm missing, I actually prefer the code that was here before; less mutation is better for little things like these IMO.

Reply:
yeah, this is a matter of "my lizard brain wants to avoid underscores that would make a reader hesitate while reading this." A thought:
```julia
s = Tuple(split(raw, ' ', keepempty=false))
first(s) == "Startdate" || return
start_raw, admin_raw, tech_raw, equip_raw = Base.tail(s)
```

Reply:
Reverted this, since tuple wrapping seems to add more ambiguity than just doing an underscore. https://github.com/beacon-biosignals/EDF.jl/pull/23/commits/ea601fed46aa23a55b16b119a9da4e752a2699a2

Reply:
Yeah, using underscore as a bit bucket variable is generally considered fine style in Julia